modules/fluid_properties/src/userobjects/TabulatedFluidProperties.C

```diff
@@ -71,14 +71,18 @@ TabulatedFluidProperties::validParams()
       true,
       "Whether pressure or temperature from tabulation exceeding user-specified bounds leads to "
       "an error.");
+  params.addParam<bool>(
+      "log_space_v",
+      false,
+      "Option to space dividing volume range in log space instead of linear space.");
```

Comment:
I suggest naming the parameter `use_log_grid_v` and changing the description to "Option to use a logarithmically-spaced grid for specific volume instead of a linearly-spaced grid."

Reply:
done